android/app/src/main/java/app/organicmaps/widget/LanesView.java

```diff
@@ -157,16 +158,16 @@ private int getAttrColor(@NonNull TypedArray data, @StyleableRes int index, @Col
 
   private void createLanesForEditMode(int lanesCount)
   {
-    final SingleLaneInfo[] lanes = new SingleLaneInfo[lanesCount];
-    lanes[0] = new SingleLaneInfo(new byte[] {1}, false);
+    final LaneInfo[] lanes = new LaneInfo[lanesCount];
+    lanes[0] = new LaneInfo(new LaneWay[] {LaneWay.ReverseLeft, LaneWay.Left}, LaneWay.None);
     if (lanes.length > 1)
-      lanes[1] = new SingleLaneInfo(new byte[] {3}, false);
+      lanes[1] = new LaneInfo(new LaneWay[] {LaneWay.SharpLeft, LaneWay.Left, LaneWay.Through}, LaneWay.None);
     for (int i = 2; i <= lanes.length - 1; i++)
-      lanes[i] = new SingleLaneInfo(new byte[] {0}, true);
+      lanes[i] = new LaneInfo(new LaneWay[] {LaneWay.Through, LaneWay.Left}, LaneWay.Through);
     if (lanes.length > 2)
-      lanes[lanes.length - 2] = new SingleLaneInfo(new byte[] {8}, false);
+      lanes[lanes.length - 2] = new LaneInfo(new LaneWay[] {LaneWay.SlightRight, LaneWay.Right}, LaneWay.SlightRight);
     if (lanes.length > 3)
-      lanes[lanes.length - 1] = new SingleLaneInfo(new byte[] {9}, false);
+      lanes[lanes.length - 1] = new LaneInfo(new LaneWay[] {LaneWay.ReverseRight}, LaneWay.None);
```

Comment:
Did you realize what the Edit mode for lanes is and what the logic behind that code is?

Reply:
Of course I realize because I'm the one who implemented it 😄 
Isn't that a beautiful and elegant way to test how the View looks like without the need to build the app, look for a road with a specific lanes and simulating navigation?

<img width="600" alt="image" src="https://github.com/user-attachments/assets/b73554c5-a56d-4d05-801e-066bb31ea7f3" />
<img width="600" alt="image" src="https://github.com/user-attachments/assets/22538edc-678e-431d-8ce3-5eea1800e2f5" />
<img width="600" alt="image" src="https://github.com/user-attachments/assets/f4f6d5f8-cb7b-4fa0-bf04-e10c24620d98" />

Reply:
Aha, adding a comment that it is used for the UI default template is ok.

I was curious, where in prod (possible) can this kind of hardcode?